src/spyglass/common/common_ephys.py

```diff
@@ -369,7 +370,9 @@ def set_lfp_electrodes(self, nwb_file_name, electrode_list):
 
         """
         # remove the session and then recreate the session and Electrode list
-        (LFPSelection() & {"nwb_file_name": nwb_file_name}).delete()
+        (LFPSelection() & {"nwb_file_name": nwb_file_name}).delete(
+            safemode=not test_mode
```

Comment:
This syntax seems odd?

Reply:
This avoids the delete prompt when in running tests. Is there something else you would prefer? Given more time to tests, I would remove the delete entirely to run this ans an insert with `replace=True`